APCAppCore/APCAppCore/DataSubstrate/Model/APCDataMigrationPolicy_Model4ToModel6.m

```diff
@@ -94,7 +93,7 @@ - (NSSet *) linkTasksToV6ScheduleMatchingV4Schedule: (id) scheduleFromV4
         NSFetchRequest *request = [APCTask requestWithPredicate: [NSPredicate predicateWithFormat: @"%K == %@ && %K == %@",
                                                                   NSStringFromSelector (@selector (taskID)),
                                                                   taskIdV6,
-                                                                  NSStringFromSelector (@selector (taskVersionNumber)),
+                                                                  @"taskVersionNumber",
```

Comment:
taskVersionNumber is no longer part of the model so the options were either to use a hardcoded string or to use a category. Since this is code for migrating from model 4 to model 6, it will only be run by someone with an older version of the app that they *haven't* been using.